cli/src/semgrep/rule_match.py

```diff
@@ -412,7 +437,13 @@ def is_blocking(self) -> bool:
         """
         blocking = "block" in self.metadata.get("dev.semgrep.actions", ["block"])
         if "sca_info" in self.extra:
-            return blocking and self.extra["sca_info"].reachable
+            if (
+                self.is_always_reachable_sca_match
```

Comment:
why are reachable transitives non-blocking? what about unknown?

Reply:
transitive reachables are blocking. transitive "always-reachable"s are non-blocking, because we think it'll be too noisy.